packages/server/lib/modes/index.ts

```diff
@@ -24,8 +21,16 @@ export = (mode, options) => {
     })
   }
 
+  const span = telemetry.startSpan({ name: `initialize:mode:${mode}` })
   const ctx = setCtx(makeDataContext({ mode: mode === 'run' ? mode : 'open', modeOptions: options }))
-  const loadingPromise = ctx.initializeMode()
+
+  telemetry.getSpan('server')?.setAttribute('mode', mode)
+
+  const loadingPromise = ctx.initializeMode().then((arg) => {
```

Comment:
is this just a `void` promise returned from the `initializeMode` function? I don't think it's called with an args right?